cmd/buildah/images.go

```diff
@@ -214,16 +215,15 @@ func outputHeader(truncate, digests bool) {
 
 func outputImages(images []storage.Image, format string, store storage.Store, filters *filterParams, argName string, hasTemplate, truncate, digests, quiet bool) error {
 	for _, image := range images {
-		imageMetadata, err := parseMetadata(image)
-		if err != nil {
-			fmt.Println(err)
-		}
-		createdTime := image.Created.Format("Jan 2, 2006 15:04")
-		digest := ""
-		if len(imageMetadata.Blobs) > 0 {
-			digest = string(imageMetadata.Blobs[0].Digest)
+		createdTime := image.Created
+
+		inspectedTime, digest, size, _ := getDateAndDigestAndSize(image, store)
+		if !inspectedTime.IsZero() {
```

Comment:
If inspectedTime is zero, then the image probably didn't include a configuration blob, since IIRC Inspect() looks there.  In that case, using the date attached to the image is probably the most correct thing, since it's either the date the image record was created locally, or (after #187 or something like it gets merged) the same value we got back from Inspect().